src/pybind/mgr/dashboard/controllers/smb.py

```diff
@@ -0,0 +1,94 @@
+
+# -*- coding: utf-8 -*-
+
+import logging
+import json
+
+from typing import Dict, List, Union
+
+
+from dashboard.controllers._permissions import CreatePermission, DeletePermission
+from smb import results
+from smb.enums import AuthMode, SMBClustering, Intent
+
+from .. import mgr
+from ..security import Scope
+from . import APIDoc, APIRouter, ReadPermission, RESTController
+
+logger = logging.getLogger('controllers.smb')
+
+
+@APIRouter('/smb/cluster', Scope.SMB)
+@APIDoc("SMB Cluster Management API", "SMB")
+class SMBCluster(RESTController):
+    _resource: str = 'ceph.smb.cluster'
+
+    @ReadPermission
+    def list(self) -> List[Dict]:
+        return mgr.remote('smb', 'show', [self._resource])
```

Comment:
res = mgr.remote('smb', 'show', [self._resource])
        if isinstance(res, list):
             return res
        return [res]

Reply:
Right, I'll update it. Thanks!